cpp/build-support/cpplint.py

```diff
@@ -4560,7 +4560,7 @@ def CheckStyle(filename, clean_lines, linenum, file_extension, nesting_state,
   CheckCommaSpacing(filename, clean_lines, linenum, error)
   CheckBracesSpacing(filename, clean_lines, linenum, nesting_state, error)
   CheckSpacingForFunctionCall(filename, clean_lines, linenum, error)
-  CheckCheck(filename, clean_lines, linenum, error)
+  #CheckCheck(filename, clean_lines, linenum, error)
```

Comment:
Is there a reason this is commented out?

Reply:
Yes, this is a lint check which fails for `DCHECK(x == y)` (consider using `DCHECK_EQ(x, y)`)